ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenKeyProvider.kt

```diff
@@ -1,9 +1,18 @@
 package org.p2p.ethereumkit.external.model
 
+import io.horizontalsystems.hdwalletkit.Mnemonic
+import org.p2p.ethereumkit.internal.core.signer.Signer
 import org.p2p.ethereumkit.internal.models.EthAddress
+import org.p2p.ethereumkit.internal.models.Chain
 import java.math.BigInteger
 
 class EthTokenKeyProvider(
-    val publicKey: EthAddress,
-    val privateKey: BigInteger
-)
+    private val seedPhrase: List<String>
+) {
+
+    private val mnemonic = Mnemonic().toSeed(seedPhrase)
+
+    val address: EthAddress = Signer.Companion.address(mnemonic, Chain.Ethereum)
+
+    val privateKey: BigInteger = Signer.privateKey(mnemonic,Chain.Ethereum)
```

Comment:
why privateKey is BigInteger? 0-0

Reply:
I don't know, but it's BigInterger from our kit